libraries/rush-lib/assets/rush-init/rush.json

```diff
@@ -42,7 +42,7 @@
    * LTS schedule: https://nodejs.org/en/about/releases/
    * LTS versions: https://nodejs.org/en/download/releases/
    */
-  "nodeSupportedVersionRange": ">=12.13.0 <13.0.0 || >=14.15.0 <15.0.0 || >=16.13.0 <17.0.0",
+  "nodeSupportedVersionRange": ">=16.13.0 <17.0.0 || >=18.12.1 <19.0.0",
```

Comment:
Are you intentionally dropping support for Node 14?  It's supported until 4/30/23.

Reply:
@mikeharder  This is just for the `rush init` template that's used when people create new Git repositories.  We could wait until the last day to adjust it, but I worried that it might take us months to remember to do that. 😆  You can of course put whatever range you like in your own **rush.json**.  

If you think it's important for some reason let me know.

Reply:
Personally, I would err on the side of allowing older Node versions beyond EOL, rather than dropping support for active Node versions.  But I agree customers can change this after `rush init` if they want, so it's not too important.